docs/maintenance/issues/Rule_Deprecations_And_Deletions.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-id: rule-deprecations
-title: Rule Deprecations
+id: rule-deprecations-and-deletions
+title: Rule Deprecations And Deletions
```

Comment:
Should it be
```suggestion
title: Rule Deprecations And Renames
```
? 
Since that's what's added elsewhere? Or 
```suggestion
title: Rule Deprecations, Renames, And Deletions
```